reference/api.html

```diff
@@ -75,7 +75,7 @@
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.1/jquery.min.js" crossorigin="anonymous"></script>
 <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.13.2/themes/smoothness/jquery-ui.css">
 <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.13.2/jquery-ui.min.js"></script>
-<script type="module" src="/assets/js/cookieconsent-init.js"></script>
+<!--<script type="module" src="/assets/js/cookieconsent-init.js"></script>-->
```

Comment:
This is the only `cookies` reference in this html (others are static, css and an image).
This also doesn't appear in the other pages, which makes me think this is the problematic line of code.